hs/KernelPerceptron/timing_drivers/KPerceptronBudgetSonar75FileIO.hs

```diff
@@ -0,0 +1,122 @@
+{-# LANGUAGE StandaloneDeriving #-}
```

Comment:
These files appear to share a lot of code. Is there a refactoring that might facilitate maintenance in the future (pulling out `print_generalization_error`, etc., into files that could be imported)?

Reply:
I did not want to have so many `FileIO` drivers for the timing analysis. The Haskell driver files rely on global variables for `n`, `m`, and `des`/`sv`. When I was working on the timing analysis drivers, I realized that the datasets have different dimensions and numbers of training/test examples. I tried to eliminate the global variables from the driver functions, but once I did so, I could not get the functions to work with each other again and compile. I could try to eliminate the global variables again, but it would involve more changes in Haskell than I may be able to complete on my own.

Reply:
Closing this pull request, but I will be working on condensing the `FileIO` drivers.